src/components/modules/auth/services/register-role.service.ts

```
@@ -0,0 +1,40 @@
+          message: "Role registered successfully",
+          data: await response.json(),
+        };
+      } else {
+        return {
+          success: false,
+          message: "Error registering role. Please try again.",
+          data: null,
+        };
+      }
+    } catch (error: unknown) {
+      if (error instanceof Error && error.hasOwnProperty("response")) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)